CHANGES/36.packaging.rst

```diff
@@ -0,0 +1 @@
+Use SPDX license identifier for project metadata.
```

Comment:
Normally, change logs shouldn't look like TODO-lists. Imperative mood isn't exactly useful to communicate changes that have already happened since the previous release. Past tense or present simple work best here:
```suggestion
The packaging metadata switched to including an SPDX license identifier introduced in :pep:`639` -- by :user:`cdce8p`.

As a side effect, the minimum required version of ``setuptools`` increased to v77.
```

One way to verify that the note looks consistent with the rest of the change log is to use the RTD preview of the PR-specific docs build @ https://aiohttp-asyncmdnsresolver--36.org.readthedocs.build/en/36/changes/.